hashicorp/boundary-ui
# ICU-14343 Updated the User Manage Dropdown button to reflect HDS Design

# Description
Updated the Manage Dropdown present on the Users scope to reflect Helios Design. Functionality is retained as previous button to add accounts to a user or delete the user entirely.


:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/14343)

## Screenshots (if appropriate)
**Before**
<img width="1869" alt="Screenshot 2024-07-09 at 11 22 00 AM" src="https://github.com/user-attachments/assets/c66ddc4a-e1e6-4a8f-b5af-75dbcd2e2f59">

**After**
<img width="2056" alt="Screenshot 2024-07-16 at 4 29 16 PM" src="https://github.com/user-attachments/assets/5f16867c-56b9-4ba6-b3ee-a7da1c73d8ed">
<img width="2054" alt="Screenshot 2024-07-16 at 4 29 26 PM" src="https://github.com/user-attachments/assets/195e8a7c-7544-4c0f-b69a-031f7775c866">



## How to Test
1. Navigate to Users Scope
2. Select a user from the table.
3. Manage dropdown should be in the top right corner. Functionality is fully retained

## Checklist


I have added before and after screenshots for UI changes
~~- [ ] I have added JSON response output for API changes~~
~~- [ ] I have added steps to reproduce and test for bug fixes in the description~~
~~- [ ] I have commented on my code, particularly in hard-to-understand areas~~
~~- [ ] My changes generate no new warnings~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
